design-system/materials/internals/definition.yaml

```diff
@@ -45,6 +45,7 @@ choiceGroupsByTheme:
         color-primary-85: 'hsl(175, 70%, 85%)'
         color-primary-90: 'hsl(175, 70%, 90%)'
         color-primary-95: 'hsl(175, 90%, 95%)'
+        color-primary-98: 'hsl(244, 100%, 99%)' # design token copied from recolouring theme for export
```

Comment:
Sorry, I don't fully understand this one. Is the idea to create a placeholder token value solely due to limitations in our tooling? I believe I would need clarification from Filip on this as I'm not familiar with the pattern.

Upon further reflection, it might be beneficial to use a placeholder value like `unused` or something similar. What do you think?

Reply:
Since `default` is the base theme, I think it should have all the available tokens.
I understand it might not needed in this case, but I think it's a good idea from the maintainability point of view.

The way I see it, I don't see the benefit in including a placeholder value but just think about what the name of the token represent and have its value in the default theme. That's why I suggest to use the proposal I shared in my previous comment.

We can have a call to discuss this further if you want to 👍

Reply:
Thanks 🙏 Still, I don't fully get the full picture and what is the proposed value for this choice, so I'd rather schedule a call to discuss it